tests/lib/rules/no-unsupported-features/es-syntax.js

```diff
@@ -492,40 +492,42 @@ runTests([
                 code: "(class { key(a = 0) {} })",
                 options: [{ version: "6.0.0" }],
             },
-            {
-                code: "function f(a = 0) {}",
-                options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
-            },
-            {
-                code: "(function(a = 0) {})",
-                options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
-            },
-            {
-                code: "((a = 0) => a)",
-                options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
-            },
-            {
-                code: "({ key(a = 0) {} })",
-                options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
-            },
-            {
-                code: "class A { key(a = 0) {} }",
-                options: [
-                    {
-                        version: "5.9.9",
-                        ignores: ["classes", "defaultParameters"],
-                    },
-                ],
-            },
-            {
-                code: "(class { key(a = 0) {} })",
-                options: [
-                    {
-                        version: "5.9.9",
-                        ignores: ["classes", "defaultParameters"],
-                    },
-                ],
-            },
+
+            // detected duplicate tests by eslint v9
+            // {
+            //     code: "function f(a = 0) {}",
+            //     options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
+            // },
+            // {
+            //     code: "(function(a = 0) {})",
+            //     options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
+            // },
+            // {
+            //     code: "((a = 0) => a)",
+            //     options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
+            // },
+            // {
+            //     code: "({ key(a = 0) {} })",
+            //     options: [{ version: "5.9.9", ignores: ["defaultParameters"] }],
+            // },
+            // {
+            //     code: "class A { key(a = 0) {} }",
+            //     options: [
+            //         {
+            //             version: "5.9.9",
+            //             ignores: ["classes", "defaultParameters"],
+            //         },
+            //     ],
+            // },
+            // {
+            //     code: "(class { key(a = 0) {} })",
+            //     options: [
+            //         {
+            //             version: "5.9.9",
+            //             ignores: ["classes", "defaultParameters"],
+            //         },
+            //     ],
+            // },
```

Comment:
Can we not leave these here commented out? 👀

Reply:
yes, just want to figure out why they are duplicated - maybe a bug somewhere. :)